.github/workflows/rust-build-and-test.yml

```diff
@@ -74,6 +75,8 @@ jobs:
       run: make test
   build:
     runs-on: ubuntu-latest
+    outputs:
+      image: ${{ steps.build.outputs.deploy_tag }}
```

Comment:
Mind renaming deploy_tag to build_tag, including the env var since this is all in the context of the build and its tests vs deployment?

Reply:
renamed